base/server/CMakeLists.txt

```diff
@@ -279,7 +279,6 @@ install(
 # install executables
 install(
     FILES
-        ${CMAKE_CURRENT_SOURCE_DIR}/sbin/pki-server-nuxwdog
```

Comment:
I don't think we can remove it in PKI 10.6. Maybe in PKI 10.7.

Reply:
Maybe we can, assuming users don't call this script directly.

Reply:
This script is NOT to be called by the user. This just generated the nuxwdog.conf file --> this is no longer required.